code/game/objects/items/stacks/cable.dm

```diff
@@ -168,6 +168,51 @@ var/global/list/datum/stack_recipe/cable_recipes = list ( \
 				getFromPool(/obj/item/stack/cable_coil, C.loc)
 				returnToPool(C)
 
+//I really wish that redundant shit wasn't needed, but snowflake code yeah
+/obj/item/stack/cable_coil/proc/catwalk_place(obj/structure/catwalk/Catwalk, mob/user)
```

Comment:
Easier is not really what I like to hear. The only check I can see which would stop placing on space is the intact check, which could be fixed by set space's intact to 0 (which it should be anyways, but isn't.) Snowflake is never better.